index.html

```diff
@@ -0,0 +1,35 @@
+
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <title>Регистрация</title>
+</head>
+<body>
+    <h1>Регистрация</h1>
+    <form method="POST" action="/register">

```

Comment:
Не знаю, как у вас, но у меня не происходило записи в БД, пока я не убрал атрибут `action`
```suggestion
    <form method="POST">

```